Makefile

```diff
@@ -1,63 +1,90 @@
 # Makefile
 
-# absolute path to the construction site
+# relative path to the construction site
 radish34=./radish34
 
 # over time, as the radish34 use-case is abstracted/generalized into
 # the baseline protocol, the pushd/popd hacks will fade away...
 
-.PHONY: build clean contracts deploy-contracts npm-install start stop reset test zk-circuits
+.PHONY: build build-containers clean contracts deploy-contracts install-config npm-install start stop system-check reset restart test zk-circuits
 
 default: build
 
 build: npm-install contracts
-	pushd ${radish34} && \
-	docker-compose build && \
-	npm run setup && \
+
+build-containers: build
+	@pushd ${radish34} && \
+	npm run dockerize && \
 	popd
 
 clean: stop
-	$(radish34)/../bin/clean_npm.sh
+	@$(radish34)/../bin/clean_npm.sh
 
 contracts:
-	pushd ${radish34} && \
+	@pushd ${radish34} && \
 	npm run build:contracts && \
 	popd
 
-deploy-contracts:
-	pushd ${radish34} && \
+deploy-contracts: install-config
+	@pushd ${radish34} && \
 	npm run deploy && \
 	popd
 
+install-config:
+	@pushd ${radish34} && \
+	npm run install-config && \
+	popd
+
 npm-install:
-	pushd ${radish34} && \
-	npm ci && npm run postinstall && \
+	@pushd ${radish34} && \
+	npm run build && \
 	popd
 
-start:
-	pushd ${radish34} && \
+start: system-check install-config build-containers
+	@pushd ${radish34} && \
 	docker-compose up -d && \
+	npm run await-stack && \
+	npm run setup-circuits && \
+	npm run deploy && \
 	popd
 
 stop:
-	pushd ${radish34} && \
+	@pushd ${radish34} && \
 	docker-compose down && \
 	popd
 
-reset:
-	pushd ${radish34} && \
-	docker-compose down && \
+system-check:
+	@npm run --silent system-check
+
+reset: stop
+	@pushd ${radish34} && \
 	docker container prune -f && \
-	docker volume rm radish34_mongo-buyer radish34_mongo-supplier1 radish34_mongo-supplier2 radish34_mongo-merkle-tree-volume radish34_chaindata && \
-	docker network rm radish34_network-buyer radish34_network-supplier1 radish34_network-supplier2 radish34_geth && \
+	docker volume rm \
+		radish34_mongo-buyer \
+		radish34_mongo-supplier1 \
+		radish34_mongo-supplier2 \
+		radish34_mongo-merkle-tree-volume \
+		radish34_chaindata || true && \
+	docker network rm \
```

Comment:
if there are any active networks on local, this errors out:`ERROR: error while removing network: network ... has active endpoints`. Probably also adding `docker volume prune -f && docker network prune -f && docker system prune -f` as an option here might help